ember-highcharts/package.json

```diff
@@ -124,6 +124,6 @@
   },
   "peerDependencies": {
     "ember-source": ">= 3.28.0",
-    "highcharts": ">= 5.0.0"
+    "highcharts": ">= 12.0.0"
```

Comment:
This would be 100% a breaking change. Not sure what would mean it wrt/ release strategy of this addon?

Reply:
Do we have to require 12 in order to get 12 to work? We couldn't do 11?

Reply:
> Do we have to require 12 in order to get 12 to work? We couldn't do 11?

Sooo real talk: Is there a _need_ to support `v11` as well? It's a [bit of boilerplate to do it](https://www.highcharts.com/docs/getting-started/version-12) and although it's doable `v12` was released 5 months ago so it feels like we could just skip?

I'm not saying "no" to the work to make v11 work, just wanted to see if it's necessary.

Thoughts @RobbieTheWagner ?

Reply:
Will take a look at the amount of work needed to make both work.

Reply:
Seems v10 is now supported as well.